React/Views/UIView+React.m

```diff
@@ -284,7 +284,13 @@ - (void)reactFocusIfNeeded {
 }
 
 - (void)reactBlur {
+#if TARGET_OS_OSX // TODO(macOS ISS#2323203)
+  if (self == self.window.firstResponder) {
```

Comment:
nit: even though upstream dot notation is used, we try to avoid dot notation in the mac fork